openshift/templates/visual-verifier/visual-verifier-deploy.bcbank.param

```diff
@@ -5,6 +5,7 @@
 #=========================================================
 NAME=visual-verifier
 APP_NAME=visual-verifier
+NAMESPACE_NAME=a99fd4
```

Comment:
Can we place this parameter after `IMAGE_NAMESPACE`, so parameters with similar meaning/context sit together?
Applies to other parameter files as well.

Reply:
param moved